hw/opentitan/ot_kmac.c

```diff
@@ -1212,6 +1200,12 @@ static void ot_kmac_regs_write(void *opaque, hwaddr addr, uint64_t value,
             qemu_log_mask(LOG_UNIMP, "%s: CMD.HASH_CNT_CLR is not supported\n",
                           __func__);
         }
+
+        if (val32 & R_CMD_ERR_PROCESSED_MASK) {
+            /* TODO: implement entropy */
```

Comment:
Comment does not apply for this field (both `entropy_req` and `hash_cnt_clr` are related to entropy stack which is not yet implemented in KMAC, `err_processed` is not)